### PR TITLE
Analyze method for FormatAnalyzer and tests

### PR DESCRIPTION
**Pull request by m09**
@vmarkovtsev's pull + a fix and some tests. WIP, tests are not complete.

### CODE REVIEW

**Comment by m09** [comment]
Travis fails on python 3.7 because of this line (46 of test_analyzer.py):

                    files[name] = File(path=name, content=bytes_content, uast=res.uast,
                                       language="JavaScript")

with error:

    TypeError: Parameter to MergeFrom() must be instance of same class: expected Node got Node. for field File.uast

And I have troubles finding why. Any idea (cc @vmarkovtsev)?

**Comment by vmarkovtsev** [comment]
@m09 OK I am taking over this PR to fix this problem.

**Comment by vmarkovtsev** [comment]
Reason: https://github.com/bblfsh/client-python/issues/122

**Comment by vmarkovtsev** [comment]
@m09 I changed the way we get UASTs - now we load them directly from disc. I intentinally made `test_analysis` fail because we need at least one comment to be generated. Please take over.

**Comment by m09** [comment]
Thanks!

File: lookout/style/format/tests/test_diff.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        cls.base_files = cls.get_files_from_tar(str(base / "freecodecamp-base.tar.xz"))
+        cls.head_files = cls.get_files_from_tar(str(base / "freecodecamp-head.tar.xz"))
+
+    def test_diff(self):
+        base_file = self.base_files["./22-index.js"]
+        head_file = self.head_files["./22-index.js"]
+        head_lines = head_file.content.decode("utf-8", "replace").splitlines()
+        new_line_indices = find_new_lines(base_file, head_file)
+        new_lines = [head_lines[i - 1] for i in new_line_indices]
+        self.assertEqual(new_lines, [
```
[the remark this answers is not in the document]
Comment (m09) [reply lookout/style/format/tests/test_diff.py]: WDYM? I inserted those lines myself

[the remark this answers is not in the document]
Comment (m09) [reply lookout/style/format/tests/test_diff.py]: I'll make it more obvious though :)

**Comment by m09** [comment]
The test to generate comments now passes but I checked quickly and in my case it generated way too many of them (one for all inputs almost). I think this is now due to the fact that the training test files make for too small a dataset and therefore produces a model that is too weak to give proper outputs because I've also tested on regular repos and commits with large trainings and there it didn't return many comments (if any).

Next step was to be decided after completion of those tests so we should decide it tomorrow (but it should probably include the creation of an anomaly/error dataset in addition to integration tests).

File: lookout/style/format/tests/test_diff.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import unittest
+
+from lookout.core.api.service_data_pb2 import File
+from lookout.style.format.diff import find_new_lines
+
+
+class AnalyzerTests(unittest.TestCase):
```
Comment (vmarkovtsev) [inline lookout/style/format/tests/test_diff.py]: Pls rename to DiffTests

**Pull request merged**